static/css/budget.less

```diff
@@ -11,30 +17,30 @@
 
       li {
           .list_item();
-          
+
           a {
               .list_link();
               .jump-link();
           }
       }
     }
-    
-    &.financial-report a, 
+
+    &.financial-report a,
     &.performance-plan a,
     &.funding-request a span{
         &:before {
             .cf-icon();
-            content: "\e402";
+            content: "\e406";
```

Comment:
Looked at the markup on these pages. These lists could and probably should be converted to using the `.list__icons` pattern, too. See: https://cfpb.github.io/cf-typography/docs/#icon-list-modifier

That would eliminate the need for a lot of this page-specific styling.